src/mixins/window-width-mixin.js

```diff
@@ -0,0 +1,34 @@
+export default {
+  data: () => {
+    return {
+      windowWidth: 0,
+      respondToResize: null,
+    }
+  },
+  mounted() {
+    this.windowWidth = window.innerWidth;
+    
+    window.addEventListener('resize', getResizeFunction());
```

Comment:
Instead of a getter, you need to reference the function directly, otherwise it's just an anonymous function, which you can't remove upon the `destroyed` hook being called.

`window.addEventListener('resize', this.handleResize);`

Reply:
Changed.